tests/zkasm/assert.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+class InstrumentInst {
+    /**
+     * @param {Object} config 
+     * @param {string} [config.outputFile] - the file to which output will be written
+     */
+    constructor(config) {
+        this.config = config;
+        this.results = {};
+    }
+
+    setup() {
+        // `zkevm-proverjs` requires a `setup` function on helper objects.
+    }
+
+    /**
+     * 
+     * @param {Object} ctx - context of zkasm program
+     * @param {Object} tag - information of helper function. 
+     * Helper function must be used in format:
+     * ${assert_eq(regname, regname, tag)}, for example:
+     * ${assert_eq(A, B, test_generated_from_line_3000)}
+     */
+    eval_assert_eq(ctx, tag) {
+        // TODO print verbose error on invalid params
+        // TODO: why arg* is list of two integers with last zero and first value in register???
+        // Is it chunks? If yes they should be merged instead of ignoring second one
+        const arg0 = ctx[tag.params[0].regName][0];
+        const arg1 = ctx[tag.params[1].regName][0];
+        const testname = tag.params[2].varName;
+        this.results[testname] = arg0 === arg1 ? 'pass' : 'fail';
+        return 0;
+    }
+
+    /**
+     * 
+     * @param {Object} ctx - context of zkasm program
+     * @param {Object} tag - information of helper function. 
+     * Helper function must be used in format:
+     * ${assert_neq(regname, regname, tag)}, for example:
+     * ${assert_neq(A, B, test_generated_from_line_3000)}
+     */
+        eval_assert_neq(ctx, tag) {
+            // TODO print verbose error on invalid params
+            // TODO: why arg* is list of two integers with last zero and first value in register???
+            // Is it chunks? If yes they should be merged instead of ignoring second one
+            const arg0 = ctx[tag.params[0].regName][0];
+            const arg1 = ctx[tag.params[1].regName][0];
+            const testname = tag.params[2].varName;
+            this.results[testname] = arg0 === arg1 ? 'fail' : 'pass';
+            return 0;
+        }
+
+    /** Writes results to `config.outputFile`. */
+    eval_assert_dump(ctx, tag) {
```

Comment:
Do we need to call `eval_assert_dump` from zkAsm for some reason?
If not, we can make it a normal method of this helper class (without `eval_` prefix) and call it in [run-tests-zkasm.js](https://github.com/near/wasmtime/pull/229/files#diff-f3af749c2e75dc7f7d3e19de136c0399d19f4ff6d23117e0a4d8a54128be6b7d) directly.

This way we can avoid the use of `fs` in this class and the need to pass the `outputFile` in the constructor and instead do all of this in `run-tests-zkasm.js`. This will make this class simpler (less responsibilities) and will make it easier to write unit-tests for it.